assets/device/classes/windowcoverings.json

```
@@ -18,7 +20,9 @@
     "sv": "Använd den här enhetsklassen för fönsterskydd när enhetsklasserna `gardiner`, `persienner` eller `solskydd` inte gäller.",
     "no": "Bruk denne enhetsklassen for vindusbeskyttelse som ikke dekkes av enhetsklassene `gardiner`, `persienner` eller `solbeskyttelse`.",
     "es": "Utiliza esta clase de dispositivo para los cobertores de ventanas que no entren en las clases 'cortinas', 'persianas' o 'toldo'.",
-    "da": "Brug denne enhedsklasse til vinduesdække, når enhedsklasserne `gardiner`, `persienner` og `markise` ikke er gældende."
+    "da": "Brug denne enhedsklasse til vinduesdække, når enhedsklasserne `gardiner`, `persienner` og `markise` ikke er gældende.",
```

```suggestion
    "da": "Brug denne enhedsklasse til vinduesdække, når enhedsklasserne `curtains`, `blinds` og `sunshade` ikke er gældende.",
```
This should also be fixed for other translations in this description